Sources/App/Views/Sections/Editor/Detail/EditorDetailViewModel.swift

```diff
@@ -177,14 +180,17 @@ final class EditorDetailViewModel: ObservableObject {
   ///                          Defaults to `nil`. Should not be `nil` if `selectedRequest` isn't.
   ///   - mode: The mode in which view should be displayed.
   ///   - completion: A closure to invoke when the user taps the `Save` button.
+  ///   - onCancel: A closure to invoke when the user taps the `Cancel` button.
   init(
     requestFile: FileSystemNode? = nil,
     requestFolder: FileSystemNode? = nil,
     requestParentFolder: FileSystemNode? = nil,
     mode: Mode = .read,
-    onSave completion: Interaction? = nil
+    onSave completion: Interaction? = nil,
+    onCancel: Interaction? = nil
```

Comment:
At this point let's remove completion.
```suggestion
    onSave: Interaction? = nil,
    onCancel: Interaction? = nil
```

update the init and the doc.